xml/ay_partitioning.xml

```diff
@@ -2566,6 +2566,104 @@ size=40G features='0' hwhandler='0' wp=rw
    </informaltable>
   </sect2>
 
+  <sect2 xml:id="ay-multidevice-btrfs">
+   <title>Multi-device Btrfs Configuration</title>
+
+   <para>
+    A <emphasis>Btrfs file system</emphasis> can be created on top of many
+    devices, offering similar features to a RAID device. With &ay;, a
+    <emphasis>multi-device Btrfs</emphasis> can be configured by specifying a
+    drive with the <literal>CT_BTRFS</literal> type. The <literal>device</literal>
+    property is used as an arbitrary name to identify each multi-device Btrfs.
```

Comment:
Why do I want that? What is the benefit of having a "multi-device Btrfs" over, let's say, a software RAID?
(Maybe it's beyond the scope of this little section to compare all possible (RAID)-technologies, but it would be nice to have at least one compelling argument for that.)

Reply:
I defer to @joseivanlopez and @ancorgs on that. I will email them and ask.

Reply:
Good question. To be honest, I don't know the real benefits of having native Btrfs RAID over classical RAID. The only thing I would point to is that having RAID in the file system itself allows you to use everything that Btrfs offers (snapshots, quotas, etc) and also RAID behavior at the same time (without mixing several technologies). Moreover, Btrfs RAID is more "versatile" because (theoretically) it supports migration from one RAID level to another and RAID levels can also be specified per file.

In my opinion, all these features are not that mature yet. For example, booting from a Btrfs RAID could be tricky. See this link for some discussion about the topic: https://unix.stackexchange.com/questions/480391/btrfs-raid-1-vs-mdadm-raid-1.

Regarding the documentation, I am not sure whether to compare RAID vs Btrfs RAID is a good idea. Maybe only commenting something about the possibility of raid level migration and raid level per file would be enough. But take into account that these advanced features are not supported by YaST/AutoYaST yet, and I don't know the plans about it.

Reply:
> To be honest, I don't know the real benefits of having native Btrfs RAID over classical RAID.

To play devils advocate here: why then including this little snippet? :joy_cat: 

> [...] Regarding the documentation, I am not sure whether to compare RAID vs Btrfs RAID is a good idea.

IMHO this is not really needed here and would probably beyond the scope of this little section. However, you answered already the question:

> The only thing I would point to is that having RAID in the file system itself allows you to use everything that Btrfs offers (snapshots, quotas, etc) and also RAID behavior at the same time (without mixing several technologies).

I think a small sentence about this would be completely sufficient.

Reply:
OK, I have added a short explanation:

>  Btrfs supports creating a single volume that spans more than one storage device, offering similar features to software RAID implementations such as the Linux kernel's built-in `mdraid` subsystem. *Multi-device Btrfs* offers advantages over some other RAID implementations. For example, you can dynamically migrate a multi-device Btrfs volume from one RAID level to another, RAID levels can be set on a per-file basis, and more. However, not all of these features are fully supported yet in SUSE Linux Enterprise Server 15 SP 2.

> With AutoYaST, a multi-device Btrfs can be configured by specifying a drive with the `CT_BTRFS` type. The `device` property is used as an arbitrary name to identify each multi-device Btrfs.